x-pack/dev-tools/jest/create_jest_config.js

```diff
@@ -40,6 +40,7 @@ export function createJestConfig({ kibanaDirectory, xPackKibanaDirectory }) {
     setupFilesAfterEnv: [
       `<rootDir>/dev-tools/jest/setup/setup_test.js`,
       `${kibanaDirectory}/src/dev/jest/setup/mocks.js`,
+      `<rootDir>/dev-tools/jest/setup/react_testing_library.js`,
```

Comment:
What is the need behind the usage of a second identical file only for x-pack? Can't we just use `${kibanaDirectory}/src/dev/jest/setup/react_testing_library.js`,

Reply:
No need! thank you